app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -443,6 +444,35 @@ export default {
         isPrivate,
       });
     },
+    onDirectFileUpload(file) {
+      if (!file) {
+        return;
+      }
+      if (checkFileSizeLimit(file, MAXIMUM_FILE_UPLOAD_SIZE)) {
+        const upload = new DirectUpload(file.file, '/rails/active_storage/direct_uploads', null, file.file.name);
+        upload.create((error, blob) => {
+          if (error) {
+            this.showAlert(
```

Comment:
@tejaswinichile Are you sure the type of error is a string?

Reply:
Yes, I checked. Let me add the screenshot for the same.